client/gutenberg/extensions/mailchimp/editor.scss

```diff
@@ -1,47 +1,21 @@
 @import './view.scss';
 
-// Variables from the https://github.com/WordPress/gutenberg to maintain style consistency with core Button block
-$big-font-size: 18px;
-$blocks-button__height: 46px;
-$blocks-button__line-height: $big-font-size + 6px;
-
 .wp-block-jetpack-mailchimp {
 
 	.wp-block-jetpack-mailchimp_notification {
 		display: block;
 	}
 
-	.wp-block-jetpack-mailchimp_button {
-		background-color: #0073aa;
-		border-radius: 5px;
-		color: var( --color-white );
-		font-family: $sans;
-		font-weight: bold;
-		margin-bottom: 1.5em;
-
-		// Styles from the Gutenberg repo to maintain style consistency with core Button block
-		border: none;
-		box-shadow: none;
-		cursor: pointer;
-		display: inline-block;
-		font-size: $big-font-size;
-		line-height: $blocks-button__line-height;
-		margin: 0;
-		padding: ($blocks-button__height - $blocks-button__line-height) / 2 24px;
-		text-align: center;
-		text-decoration: none;
-		white-space: normal;
-		word-break: break-all;
-		.editor-rich-text__tinymce {
-			cursor: text;
-		}
-	}
-
 	.editor-rich-text__inline-toolbar {
 		pointer-events: none;
-		.components-toolbar  {
+		.components-toolbar {
 			pointer-events: all;
 		}
 	}
 
+	// Hide everything else except notification when modifying notification labels
+	&.wp-block-jetpack-mailchimp_notication-audition > *:not( .wp-block-jetpack-mailchimp_notification ) {
```

Comment:
I had to add this after I removed `p` tags that were causing React rendering issues.

Previously we had `&.wp-block-jetpack-mailchimp_notication-audition p` here.